blog/2022/storage-map.adoc

```diff
@@ -0,0 +1,89 @@
+:title: New storage in Keycloak
+:date: 2022-07-28
+:publish: true
+:author: Hynek Mlnařík
+
+In 2021, Keycloak team started implementing a new storage capable
+of upgrading Keycloak without downtime. We are happy to announce that
+with Keycloak 19, the preview of this storage is available in two 
+implementations: relational database and Infinispan.
```

Comment:
There was/is much more reasons for the new store than zero downtime upgrade support.

I'd call it an early preview, rather than a preview. Question is would we want to combine this blog post and the store road-map blog post? That might make sense, as you could relatively easily include in this post what's coming in 20, when it's aimed to be production ready, etc..